bundles/org.openhab.binding.ism8/src/main/resources/OH-INF/addon/addon.xml

```diff
@@ -4,7 +4,7 @@
 	xsi:schemaLocation="https://openhab.org/schemas/addon/v1.0.0 https://openhab.org/schemas/addon-1.0.0.xsd">
 
 	<type>binding</type>
-	<name>ISM8 Binding</name>
+	<name>Wolf ISM8 Binding</name>
```

Comment:
Please regenerate the I18N properties file after this change:
https://github.com/openhab/openhab-addons/blob/772026e25e304127a727112fe48b4f3f2c15383d/bundles/org.openhab.binding.ism8/src/main/resources/OH-INF/i18n/ism8.properties#L3

Also, when renaming the binding, for consistency it should probably also be renamed in these places:
https://github.com/openhab/openhab-addons/tree/main/bundles/org.openhab.binding.ism8#ism8-binding (the headline)

https://github.com/openhab/openhab-addons/blob/772026e25e304127a727112fe48b4f3f2c15383d/bundles/org.openhab.binding.ism8/pom.xml#L15

https://github.com/openhab/openhab-addons/blob/772026e25e304127a727112fe48b4f3f2c15383d/bundles/org.openhab.binding.ism8/src/main/feature/feature.xml#L5

Reply:
I am not sure if this is the right approach. I think that "renaming" the binding may be too much - just to give a better search experience.
Maybe it would be better to extend the search in the UI from title to the descriptions....
...and if that is too much effort, even living with the current situation might be an option.

What are your thoughts on that? @jlaur @lsiepel?

Reply:
I agree that bindings should not be renamed just to add search words for the UI, so if the renaming otherwise doesn't make sense or seems overkill, then I would also rather prefer one of your other suggested options.

I don't know if it is feasible to use description for the search as well, but you could try to open an issue in the openhab-webui repo. At first it seems straightforward, but when looking at a description, for example this one: "This is the binding for the ISM8 card used for Wolf heating systems or other Wolf eBus devices.", I wonder if certain words should be excluded to not give too many unrelated results. At least it looks like a simple substring matcher, but maybe there is a fancy search machine behind. 😄